tools/buddy-opt/buddy-opt.cpp

```diff
@@ -80,6 +80,10 @@ void registerLowerSchePass();
 void registerFuncBufferizeDynamicOffsetPass();
 void registerConvertMemcpyToGPUPass();
 void registerLegalizeShmemOutliningPass();
+void registerMatMul_TransposeB_VecPass();
```

Comment:
What is the difference between `registerMatMul_TransposeB_VecPass` and `registerMatMulTransposeBVecPass`? Does this code have any particular meaning?

Reply:
sorry,i typed an extra function.there is no pass named  "registerMatMul_TransposeB_VecPass"

Reply:
I'll delete it